omod-1.8/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/SystemInformationResource1_8Test.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs1_8;
+
+import static org.junit.Assert.assertEquals;
+import javax.servlet.http.HttpServletResponse;
+import org.apache.struts.mock.MockHttpServletRequest;
+import org.apache.struts.mock.MockHttpServletResponse;
+import org.apache.commons.beanutils.PropertyUtils;
+import org.junit.Assert;
+import org.junit.Test;
+import org.openmrs.api.impl.AdministrationServiceImpl;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.webservices.rest.SimpleObject;
+import org.openmrs.module.webservices.rest.test.Util;
+import org.openmrs.module.webservices.rest.web.v1_0.controller.MainResourceController;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.openmrs.web.test.BaseModuleWebContextSensitiveTest;
+
+public class SystemInformationResource1_8Test extends BaseModuleWebContextSensitiveTest {
+	
+	@Autowired
+	private MainResourceController mainResourceController;
+	
+	@Test
+	public void testGetAll() throws Exception {
+		SimpleObject rest = new SimpleObject();
+		rest.put("SystemInfo", Context.getAdministrationService().getSystemInformation());
+		MockHttpServletResponse response = new MockHttpServletResponse();
```

Comment:
@dkayiwa  I am working with test cases, I couldn't implement those type of test cases here, Give me one more day to finalize